nxc/modules/schtask_as.py

```diff
@@ -207,14 +230,12 @@ def execute_handler(self, command, fileless=False):
 
         dce.set_credentials(*self.__rpctransport.get_credentials())
         dce.connect()
-
-        tmpName = gen_random_string(8)
-
+        tmpName = gen_random_string(8) if self.task is None else self.task
         xml = self.gen_xml(command, fileless)
 
         self.logger.info(f"Task XML: {xml}")
         taskCreated = False
-        self.logger.info(f"Creating task \\{tmpName}")
+        self.logger.display(f"Creating task \\{tmpName}")
```

Comment:
Imo this is a bit spammy, I would leave it at logging as info.

Reply:
Personally like seeing this as output,  I could add a debug flag to see this type of info.

Reply:
You can also use `--verbose` to only have the info logs without all the debug logs if you would like to have a bit more information.